### ISSUE TITLE
Complete sample for in-memory implementation

### ISSUE THREAD

**Issue by RPM1984** [json-api-dotnet/JsonApiDotNetCore#372]
Hi there 👋 

Any chance of getting a complete sample for a basic in-memory implementation?

Just trying to get a simple API up and running using in-memory objects, but not having much luck.

I started off the [NoEntityFrameworkExample](https://github.com/json-api-dotnet/JsonApiDotNetCore/tree/master/src/Examples/NoEntityFrameworkExample), which runs fine. But now i'm trying to go _one little step_ further, and include related entities. This (i'm assuming) requires the `GetRelationshipsAsync` methods implemented... but i have no idea how to do that 😞 

I've tried.. but struggled, so i'm guessing others might too?

So basically - any chance of finishing the NoEntityFrameworkExample to include relationships, or have another simple in-memory one?

Failing that - does anyone have an example of a relevant sample elsewhere, that i can reference?

Many thanks!

**Comment by jaredcnance** [comment]
> in-memory objects

Can you be a little more specific about what you're trying to do? i.e.

- Do you want to maintain an in-memory store that supports all json:api operations? If so, have you thought about using EF Core's in memory database? If that doesn't work, can you elaborate on why?
- Or are you simply wanting to serve static data from a json:api endpoint?

**Comment by RPM1984** [comment]
Hi @jaredcnance .. wow quick reply! 😄 

I'm not trying to do anything 'realistic' yet... just wanted to get some fake data up and running (so yes - static data), in order to demonstrate the concepts quickly (prototype).

I haven't tried in-memory EF, i guess that'll work. I just started with the 'no entity framework' sample, but got stuck. I'll try out the in-memory EF one. Thanks!
